test/cypress/integration/payment-links/delete-payment-links.cy.test.js

```diff
@@ -1,24 +1,37 @@
 const gatewayAccountStubs = require('../../stubs/gateway-account-stubs')
 const userStubs = require('../../stubs/user-stubs')
-const { getProductsByGatewayAccountIdAndTypeStub, getProductByExternalIdStub, deleteProductStub } = require('../../stubs/products-stubs')
+const {
+  getProductsByGatewayAccountIdAndTypeStub,
+  getProductByExternalIdStub,
+  deleteProductStub
+} = require('../../stubs/products-stubs')
+const { deleteTokenByApiTokenSuccess } = require('../../stubs/token-stubs')
 const userExternalId = 'a-user-id'
 const gatewayAccountExternalId = 'a-valid-account-id'
 const gatewayAccountId = 42
 const productExternalId = 'a-product-id'
+const apiToken = 'an-api-token'
 
 const product = {
-  external_id: productExternalId
+  external_id: productExternalId,
+  payApiToke: apiToken
```

Comment:
typo - payAPIToken

Reply:
Thanks, seems I got interrupted part way through adding the missing stub to this test and it wouldn't have worked. Have pushed e6759f5fb6610ec2188dfb9bfcd8fd97e4c45a08 which fixes this test.

Will squash all the commits in this PR before merging